src/javascript/app/pages/user/new_account/welcome_page.js

```diff
@@ -1,20 +1,22 @@
-const BinarySocket   = require('../../../base/socket');
-const Client         = require('../../../base/client');
-const localize       = require('../../../../_common/localize').localize;
-const createElement  = require('../../../../_common/utility').createElement;
-const getElementById = require('../../../../_common/common_functions').getElementById;
-const Url            = require('../../../../_common/url');
+const BinarySocket      = require('../../../base/socket');
```

Comment:
@brandon-binary for each import statement there is an extra space before equal sign (=)

Reply:
@nazaninreihani what do you mean?